website/widgets/form/table.md

```diff
@@ -58,19 +58,18 @@ q.page['example'] = ui.form_card(box='1 1 5 3', items=[
 
 ## With selection
 
-If `multiple` is set to False (default), each row in the table is clickable. When a cell in the column with `link=True`
+By default, each row in the table is clickable. When a cell in the column with `link=True`
 (defaults to first column) is clicked or the row is doubleclicked, the form is
 submitted automatically and `q.args.table_name` is set to `[row_name]`, where
 `table_name` is the `name` of the table, and `row_name` is the `name` of the row that was clicked.
 Note that doubleclicking the row if single selection is active results in submission as well.
 
-If `multiple` is set to `True`, each row in the table is selectable. A row can be selected by
-clicking on it.
-Multiple rows can be selected either by shift+clicking or using marquee selection. When the form
-is submitted, `q.args.table_name` is set to `[row1_name, row2_name, ...]` where `table_name` is the
-`name` of the table, and `row1_name`, `row2_name` are the `name` of the rows that were selected. Note
-that if `multiple` is set to `True`, the form is not submitted automatically and one or more buttons in the form are
-required to trigger submission.
+However, this behavior changes when the multiple or single selection is activated (`multiple` or `isSingle` is set to `True`). In this case each row in the table becomes selectable.
+Also the form is not submitted automatically and one or more buttons in the form are required to trigger submission.
```

Comment:
Is this tested?

Reply:
When `multiple` or `single` is set to `True`, double click is not submitting the form, but when clicking the cells in column with `link=True` (first column by default), the submit is triggered, which should not happen according to [our current docs](https://wave.h2o.ai/docs/widgets/form/table/#with-selection):

> Note that if multiple is set to True, the form is not submitted automatically and one or more buttons in the form are required to trigger submission.

...but it happens also in the master. That means we should handle this case and [fire onClick logic](https://github.com/h2oai/wave/blob/cc2d18dee0d6784279311f7f6d437145425b056e/ui/src/table.tsx#L512) only when `!(isMultiple && !isSingle)`. We should either edit the implementation to match what is described in our current docs or to change our docs. I think changing the implementation is better solution here. It makes no sense that you can submit multiple selected items by clicking on any of the table cells in column with `link=True`.

Reply:
> It makes no sense that you can submit multiple selected items by clicking on any of the table cells in column with link=True

This would indeed make no sense, but it's not what would happen anyway. If you click on a link True col, only the single clicked rowname shall be submitted, which is correct. The docs are technically correct as well but might be confusing. It would be better if they were clear that checkmark selection (single/multi) can only be submitted via external button or something (useful for making a delete action for example) and clicking the row will always submit only the corresponding row - e.g. go to detail.

Reply:
Now it should be clear:
![image](https://user-images.githubusercontent.com/23740173/215456938-3739f29d-1de4-42df-acdf-ab9e5dffc36d.png)